slider-entity-row.js

```diff
@@ -204,7 +204,10 @@ class SliderEntityRow extends Polymer.Element {
           return stateObj.attributes.speed;
         },
         min: (stateObj) => 0,
-        max: (stateObj) => stateObj.attributes.speed_list.length - 1,
+        max: (stateObj) => {
+          this.controller.valueDelta = stateObj.attributes.speed_list[0].toLowerCase() === 'off' ? 0 : 1;
```

Comment:
This means `set` and `get` will fail if `max` hasn't been called first. While that's always true now, it's annoying to maintain...

Reply:
Hmm that's a good point. I could see that causing problems...     
I could set `valueDelta: 0` in the object there, but I would rather keep it similar to the other controller objects. I would prefer that to adding if statements all over the place though.